app/scripts/filters/util.js

```diff
@@ -348,6 +348,30 @@ angular.module('openshiftConsole')
           .replace(/^./, function(str){ return str.toUpperCase(); });
     };
   })
+  // event messages should simple, but human readable:
+  // Pulled Pod <name>
+  // Unhealthy Pod <name>
+  // Deployment created for DeploymentConfig <name>
+  // etc.
+  // This filter handles the "Reason <optional preposition>"
+  // but not the <name> as we <a> link that in templates.
+  .filter('messageForEvent', function($filter) {
```

Comment:
Example with the `&mdash;`:

![screen shot 2017-08-16 at 10 22 39 am](https://user-images.githubusercontent.com/280512/29368159-ec6fed1a-826c-11e7-8192-9616c96c12a4.png)